core/file_walker.go

```diff
@@ -60,9 +60,12 @@ func GetSubTree(path string, parent *File, readDir ReadDir, ignoredFolders map[s
 			files = append(files, file)
 		}
 	}
-	ret.Name = name
 	if parent != nil {
+		ret.Name = name
 		ret.Parent = parent
+	} else {
+		// Root dir
+		ret.Name = dirName + name
```

Comment:
By going out of parsed tree I mean that you would see folders (e.g. `/home`) when browsing through folders in UI, but you haven't actually parse their content